### ISSUE TITLE
Default Keyboard Shortcut

### ISSUE THREAD

**Issue by ghost** [subgraph/gnome-shell-extension-torstatus#16]
Evening,

I've recently caught myself being annoyed with having to use the torstatus extension GUI in order to build a new Tor Identity. Dragging the mouse to the edge of the screen, clicking to open the dropdown menu, dragging down towards "Tor Network", clicking to open another dropdown menu that reveals a sole option "New Identity", clicking in it. Such a recurrent action, it should be reduced to a single keystroke, I thought.

Hence, with the aid of dawuud at Subgraph's IRC channel, I've built a script in order to do that and assigned it to a keybinding.

This is the script (dependencies include `expect`, `telnet` and `notify-send`):
```
#!/bin/bash
# Build new Tor circuit and send a desktop notification upon completion

Icon="/usr/share/gnome-shell/extensions/torstatus@subgraph.com/icons/tor-simple-symbolic.svg"
Title="Tor Network"
BodyText="Switched to a new Tor identity!"

expect -c '
	spawn	telnet 127.0.0.1 9051
	expect	"Escape character is '^]'."
	send	"authenticate\r"
	expect	"250 OK"
	send	"SIGNAL NEWNYM\r"
	expect	"250 OK"
	close'
notify-send -i "$Icon" "$Title" "$BodyText"
```
And its respective dconf dump:
```
[custom-keybindings/custom1]
binding='<Super>Print'
command='bin/build-tor-circuit'
name='Build new Tor circuit'
```
In conclusion, I suggest as an enhancement a default keyboard shortcut to build a new Tor cirtcuit. As a reference of a gnome-shell extension with keyboard shortcuts set by default, and also an easy to use configuration menu for the tweak tool, one could look at other gnome-shell extensions such as [shellshape](https://github.com/timbertson/shellshape).

Best of luck.
